packages/passport/sdk/src/Passport.ts

```diff
@@ -134,18 +135,29 @@ export class Passport {
    * `connectImx` instead.
    */
   public async connectImxSilent(): Promise<IMXProvider | null> {
-    return this.passportImxProviderFactory.getProviderSilent();
+    const flow = trackFlow('passport', 'connectImxSilent');
+    flow.addEvent('startConnectImxSilent');
+    const provider = this.passportImxProviderFactory.getProviderSilent();
+    flow.addEvent('endConnectImxSilent');
+    return provider;
   }
 
   public async connectImx(): Promise<IMXProvider> {
-    return this.passportImxProviderFactory.getProvider();
+    const flow = trackFlow('passport', 'connectImx');
+    flow.addEvent('startConnectImx');
+    const provider = this.passportImxProviderFactory.getProvider();
+    flow.addEvent('endConnectImx');
```

Comment:
Can we follow the same pattern that we're using in `zkEvmProvider.ts` please? E.g.:
```
const flow = trackFlow('passport', FLOW_NAME);
try {
} catch (error) {
  if (error instanceof Error) {
    trackError('passport', FLOW_NAME, error);
  }
  flow.addEvent('errored');
  throw error;
} finally { 
  flow.addEvent('end');
}
```